### DISCUSSION TITLE
Registered api working good locally and 404 from any remote machine - Need help to resolve

### DISCUSSION THREAD

**Discussion by MalarVelu** [apiman/apiman#2574] [General]
We registered an api in apiman successfully and working as expected locally. But while connecting the api from any other machine is throwing 404 error. api is having no policay. I tried including CORS policy for allowing cross origins which didn't work either.

**Comment by msavy** [comment]
This is not enough information to understand what your problem is. It could be your firewall configuration or network configuration.
